packages/gen-ai-hub/src/index.ts

```diff
@@ -1,10 +1,80 @@
-export * from './client/index.js';
+export type {
+  OpenAiChatModel,
```

Comment:
Did we check what part of this API we actually need to expose? Or is this simply exporting everything, but named?

Reply:
This is exporting everything that is not marked internal.

Reply:
I'm not sure if we followed the @internal convention so far though :D for now this is fine, could you create a follow-up ticket to check if everything is properly tagged with @internal? And also make this a pre-requisite for the GA ticket.

Reply:
I think for the AI sdk, we will just be enforcing that everything that is not exported through the public api is marked with @internal. The added script already checks for this.

Reply:
Yes, after the script runs on every PR and forces everyone to reflect if an API should be internal or public this makes sense.
Currently, though, this convention was not consistently followed, and with your PR you are adding every untagged API to the public API, even though this might not be intentional.

Reply:
Therefore we should at least check every now "public" API to make sure they should genuinely be public before GA.
After that this should never be necessary again.

Reply:
Ah okay I got what your mean now. I'll create a follow up to properly check for @internal consistency.